### ISSUE TITLE
Question - Depth | Batch From Directory - Existing DepthMaps, Are they Regenerated?

### ISSUE THREAD

**Issue by wcapes** [thygate/stable-diffusion-webui-depthmap-script#403]
Hi, just a query, maybe I'm stupid and missed something

Here are my settings, and I'm using the Depth | Batch From Directory option 

![image](https://github.com/thygate/stable-diffusion-webui-depthmap-script/assets/88830951/054d4f4d-1fc5-4ab4-92b1-34d2e118e7e8)

I have 2000 images in my Input Directory that I set, and 2000 depth maps in my Output Directory that I set
When I run, the log does show that it picks up the Source and the Custom Depth Maps

So basically I'm trying to take an image, with a depth map and create a SBS image for 3D.

Does this regenerate the depthmap?   As 2000 iamges takes 60 minutes on a Ryzen 5 5500 , Geforce 1660 Super card ( Yes I know it's the low end ) but an hour to generate 2000 images when the depth is already created, seems I must be missing something

Please help :)

**Comment by semjon00** [comment]
Well, try experimenting with smaller amount of images. If you generate from folder, and there is a file named "aaa.png", then the resulting depthmap will be saved into new folder as "aaa-0000.png". If you do select the "skip generation" checkmark, this file will be reused.

**Comment by semjon00** [comment]
To reuse depthmaps, you need to generate and save them first.

**Comment by semjon00** [comment]
A better description in wiki would be good for that.
